v22.1/unsplit-at.md

```diff
@@ -214,6 +100,8 @@ Now unsplit the index to remove the split enforcements:
 (3 rows)
 ~~~
 
+You can see the split's expiration date in the `split_enforced_until` column. The [`crdb_internal.ranges`](crdb-internal.html) table also contains information about ranges in your CockroachDB cluster, including the `split_enforced_until` column.
```

Comment:
`crdb_internal` is marked as a `Reserved` interface in the [support policy](https://www.cockroachlabs.com/docs/v22.1/api-support-policy.html), and the `ranges` table is not listed as one of the "ok for production" exceptions in the `crdb_internal` doc. IMO we should not bring it up, but happy to defer to any eng who disagrees.

```suggestion
You can see the split's expiration date in the `split_enforced_until` column.
```

Reply:
I asked @kevin-v-ngo about that and he said it was OK to leave it in.